SimG4Core/Application/src/ParametrisedEMPhysics.cc

```diff
@@ -162,11 +164,11 @@ void ParametrisedEMPhysics::ConstructProcess() {
     }
 
     if (gem || gemHad) {
-      G4Region* aRegion = G4RegionStore::GetInstance()->GetRegion("EcalRegion");
+      G4Region* aRegion = G4RegionStore::GetInstance()->GetRegion("EcalRegion", false);
 
       if (!aRegion) {
-        edm::LogVerbatim("SimG4CoreApplication") << "ParametrisedEMPhysics::ConstructProcess: "
-                                                 << "EcalRegion is not defined, GFlash will not be enabled for ECAL!";
+        edm::LogWarning("SimG4CoreApplication") << "ParametrisedEMPhysics::ConstructProcess: "
```

Comment:
@civanch this is quite a jump in verbosity, do we really need to treat it as a warning? As this code is devoted to GFlash, if GFlash is not activated in ECAL probably it makes sense...

Reply:
@fabiocos , this warning is valid: if GFlash is asked for EcalRegion but EcalRegion is not defined the warning should be seen.